meinberlin/apps/kiezradar/serializers.py

```diff
@@ -102,4 +105,15 @@ def to_representation(self, instance):
             {"id": topic.id, "name": topics_enum(topic.code).label}
             for topic in instance.topics.all()
         ]
+        if not instance.name:
+            representation["name"] = _("Searchprofile %d") % instance.number
         return representation
+
+    def _set_number(self, validated_data):
```

Comment:
I'm adding the number here, but now unsure if in the models `save()` is better, what do you think?